### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xrdavies**
Add one more tip to run integrationTest locally

### CODE REVIEW

**Comment by pmprete** [comment]
Just curious where are you deploying this? if you use development or rskregtest it should work fine.
If you are using ethereum or rsk testnet or mainnet the ERC1820 is already deployed, so i don't see a reason why you should check the name on the deployment step

**Comment by xrdavies** [comment]
> Just curious where are you deploying this? if you use development or rskregtest it should work fine.
> If you are using ethereum or rsk testnet or mainnet the ERC1820 is already deployed, so i don't see a reason why you should check the name on the deployment step

Good question!

I'm running it with my local dev net - two ganache networks. To get test coins on Ethereum testnet is not so friendly so I change to use local dev nets. 

Currently I'm working on top of your code to support other EVM like chains rather than rsk <-> Ethereum.

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ An integration test is prepared for contracts and federators. To properly run in
 1. Check `infura.key` in `bridge`
 1. Check `federator.key` in `federator/config`
 1. Check `config.js` in `federator/config`
+1. Check your `networkName` in `bridge/migrations/4_deploy_erc1820.js`
```
Comment (pmprete) [inline README.md:59]: Add something like  "that this step is necessary only if the network you are working on does not have ERC1820 deployed"  and i'll merge it

**Review by pmprete — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xrdavies) [reply README.md:59]: Okay.

Comment (xrdavies) [reply README.md]: Done. Please help to review this PR.

**Review by pmprete — APPROVED** [review]
Thanks @xrdavies

**Pull request merged**